osf/models/node.py

```diff
@@ -2813,8 +2815,15 @@ def rename_node_wiki(self, name, new_name, auth):
             save=True,
         )
 
-    def delete_node_wiki(self, name, auth):
-        page = self.get_wiki_page(name)
+        return page
+
+    def delete_node_wiki(self, name_or_page, auth):
```

Comment:
See previous comment: this should probably go on `WikiPage`. We want to remove as much wiki-related code from the node as possible.